src/Turbo/.gitattributes

```diff
@@ -0,0 +1,10 @@
+/.github export-ignore
+/.php_cs.dist export-ignore
+/assets/test export-ignore
+/tests/ export-ignore
+/phpstan.neon export-ignore
+/phpunit.xml.dist export-ignore
+/.gitignore export-ignore
+/.gitattributes export-ignore
+src/Resources/assets/package-lock.json
+src/Resources/assets/node_modules/
```

Comment:
What do these lines do?

Reply:
This file isn't needed at all actually. I remove it.